README.md

```diff
@@ -59,6 +59,11 @@ _Other installation options (Homebrew, Docker, etc.) can be found [here](https:/
 
 `helm plugin install https://github.com/datreeio/helm-datree`  
 
+## 🗂 Kustomize support
+Datree comes with out-of-the-box [support for Kustomize](https://hub.datree.io/kustomize-support/#utm_source=github&utm_medium=organic_oss):
+
+`datree kustomize test [path/to/kustomization.yam/dir/]`
```

Comment:
".yam" should be ".yaml"

I think that this is clearer:
`datree kustomize test [path to kustomization.yaml dir]`